envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadataValidatorTest.kt

```diff
@@ -62,11 +83,62 @@ class NodeMetadataValidatorTest {
         )
         val request = DiscoveryRequest.newBuilder().setNode(node).build()
 
-        // when
-        permissionsDisabledValidator.onStreamRequest(123, request = request)
+        // then
+        assertDoesNotThrow { permissionsDisabledValidator.onStreamRequest(123, request = request) }
+    }
+
+    @ParameterizedTest
+    @MethodSource("configurationModeNotSupported")
```

Comment:
Have you thought about `@CsvSource` parameter injection? With current approach it's difficult to read the arguments as I have to jump between method declaration and the test definition to map which bool arg corresponds to which flag.

Reply:
I didn't know about that. I've changed to `@CsvSource.`